tests/queries/0_stateless/02662_sorting_constants_and_distinct_crash.sql

```diff
@@ -0,0 +1,25 @@
+drop table if exists test_table;
+CREATE TABLE test_table (string_value String) ENGINE = MergeTree ORDER BY string_value;
+insert into test_table select * from (
```

Comment:
Probably make sense to add `SYSTEM STOP MERGES test_table`. With some comment. Since we couldn't reproduce it after merges happened. I can be not necessary for the test, but will provide full info to future readers